libenkf/src/enkf_config_node.c

```diff
@@ -410,6 +410,26 @@ enkf_config_node_type * enkf_config_node_alloc_GEN_PARAM( const char * node_key
 }
 
 
+enkf_config_node_type * enkf_config_node_alloc_GEN_DATA_everest( const char * key ,
+                                                                 const char * result_file_fmt,
+                                                                 const int_vector_type * report_steps) {
+
+  if (!gen_data_config_valid_result_format( result_file_fmt )) {
+    printf("RETURN NULL\n");
```

Comment:
Debug statement? I suggest we turn it into a proper res_log call or remove it..

Reply:
> I suggest we turn it into a proper res_log call or remove it..

Removed